### ISSUE TITLE
feedback

### ISSUE THREAD

**Issue by gitlerat** [kognise/water.css#296]
Hi, I think its not recommended using px so much, most times specially fonts it is recommended to use rem.
Also please consider adding some modality on the css file like https://classless.de/ is doing. Also some things classless does is really top, maybe you can have  look :)

**Comment by LoganDark** [comment]
Absolutely, `rem` is the way to go now, there's a reason Tailwind CSS is built entirely on the `0.25rem` scale.

**Comment by deepbeatz** [comment]
@gitlerat please assign me this issue. I want to start working on it immediately.

**Comment by gitlerat** [comment]
I think I have no write access on this repo because I cant assign the issue to you :/

**Comment by Nikhil743** [comment]
please assign me this issue. I want to start working on it immediately.
